### PR TITLE
Setup minikube with internal registry 

### PR DESCRIPTION
**Pull request by Frawless**
Signed-off-by: Jakub Stejskal <xstejs24@gmail.com>

### Type of change

_Select the type of your PR_

- Enhancement / new feature

### Description

This PR changes the vmdriver for minikube to `docker`, which allows us to setup a local registry for testing KafkaConnect build feature on minikube cluster.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

**Review by scholzj — APPROVED** [review]

File: systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java [hunk 1]
```diff
@@ -370,5 +369,19 @@ void setup() {
         installClusterOperator(NAMESPACE, Constants.CO_OPERATION_TIMEOUT_SHORT);
 
         KafkaResource.createAndWaitForReadiness(KafkaResource.kafkaEphemeral(clusterName, 3).build());
+
+        KafkaResource.createAndWaitForReadiness(KafkaResource.kafkaEphemeral(clusterName, 3).build());
```
Comment (im-konge) [inline systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java:373]: Maybe the GH shows me this wrong, but are there two Kafka clusters?

**Review by im-konge — COMMENTED** [review]
Just one comment, thank you for this :slightly_smiling_face:

**Review by im-konge — APPROVED** [review]

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Comment by Frawless** [comment]
@strimzi-ci run tests test_only profile=oauth

**Pull request merged**